src/scripts/main.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+const body = document.querySelector('body');
+const createElement = function (className, text) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+
+  div.className = className;
+
+  div.textContent = text;
+
+  body.append(div);
+};
+let isClicked = false;
+let leftClick = false;
+const firstPromise = new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    isClicked = true;
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    if (!isClicked) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  body.addEventListener('click', () => {
+    resolve('Second promise was resolved');
```

Comment:
Remove duplicates everywhere
```suggestion
    resolve();
```